### PR TITLE
Properly hides assets from dashboard when they are in the ignore list

### LINKED ISSUE #1501: Allow ability to hide/ignore balances of some coins
## Abstract

My ethereum address got sent some tokens that I don't care about. They are worth pennies. Please add the ability to hide and ignore these tokens so it won't show up in my dashboard.

## Motivation

This will make the dashboard much cleaner. Maybe just add a right click to hide coin option. And a settings to manage hidden coins. You may even reuse the settings in accounting section that lets you ignore coins for accounting.

## Specification

If the feature is techical in nature please write as detailed as possible a specification of what needs to be built.

### PR DESCRIPTION
**Pull request by kelsos**
Closes #1501

### CODE REVIEW

File: frontend/app/src/locales/en.json [hunk 1]
```diff
@@ -644,5 +664,46 @@
...
+      "labels": {
+        "ignore": "Select asset to ignore",
+        "unignore": "Select asset to remove from ignored assets",
+        "unignore_hint": "Click to see all ignored assets and select one for removal"
+      },
+      "ignore_tags_hint": "Click to see all assets and select one to ignore"
+    },
+    "validation": {
+      "tax_free_days": "Please enter the number of days",
+      "tax_free_days_gt_zero": "The number of days cannot negative or zero"
```
Comment (LefterisJP) [inline frontend/app/src/locales/en.json:693]: cannot negative -> cannot be negative

**Review by LefterisJP — COMMENTED** [review]
I tested it and it works. But it just works for the dashboard.

Shouldn't this be also extended to:

Blockchain balances and exchange balances and manual balances? (okay last one may not make sense -- but the rest do imo)

**Review by LefterisJP — CHANGES_REQUESTED** [review]
@kelsos There is a serious problem introduced by this PR. I can no longer expand ethereum accounts and see what assets each one owns

**Comment by kelsos** [comment]
OK let me check it the fix should be simple

**Review by LefterisJP — APPROVED** [review]
LGTM now

**Comment by kelsos** [comment]
@LefterisJP backend tests failed on travis, I am merging

**Comment by LefterisJP** [comment]
Hmm please don't merge without explaining why you chose to ignore the failing of the backend tests. For example you must have seen I always check if it's a problem with one of the remotes the test may have failed to contact e.t.c.

I made an issue for the failure: https://github.com/rotki/rotki/issues/1550

**Comment by kelsos** [comment]
@LefterisJP Well the reasoning is simple, this PR is only changing the frontend of the application. Regarding the mergability of the PR, it doesn't really matter which backend test failed and why. Does it? There was no modification of the python code, which means that whatever the problem, was not introduced by this PR. 

Furthermore, I just pinged you since it didn't seem like a remote service issue :) In the future, I will try adding more information in regards to the type of failure.

**Pull request merged**